PR: https://github.com/odoo/odoo/pull/

From: 7978d4ba0f093c4b5400dbb1776ff5c0416e203b
From: Yannick Tivisse
Date: 2018-09-10 12:23:42

Structural Changes: 4
Total Changes: 22

Revert "[IMP] *: Define groups on res.users models"

This reverts commit 99f497b390e680629dce3a7e2e282ada74ee9884.

================================= pseudo patch: =================================

--- a/addons/project/models/__init__.py
+++ b/addons/project/models/__init__.py
@@ -5,4 +5,3 @@ from . import project
 from . import res_config_settings
 from . import res_company
 from . import res_partner
-from . import res_users

--- a/addons/project/models/res_users.py
+++ b/None
@@ -1,21 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import fields, models
-
-
-class ResUsers(models.Model):
-    _inherit = 'res.users'
-
-    has_group_subtask_project = fields.Boolean(
-        'Use Subtasks', compute='_compute_groups_id', inverse='_inverse_groups_id',
-        group_xml_id='project.group_subtask_project')
-
-    has_group_project_rating = fields.Boolean(
-        'Use Rating on Project', compute='_compute_groups_id', inverse='_inverse_groups_id',
-        group_xml_id='project.group_project_rating')
-
-    group_project_user = fields.Selection(
-        selection=lambda self: self._get_group_selection('base.module_category_project_management'),
-        string='Project', compute='_compute_groups_id', inverse='_inverse_groups_id',
-        category_xml_id='base.module_category_project_management')
